frontend/src/components/sidebar/SideBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Drawer from "react-modern-drawer";
+import "react-modern-drawer/dist/index.css";
+import { NavLink } from "react-router-dom";
+import classes from "./SideBar.module.scss";
+import { ReactComponent as SettingsIcon } from "../../assets/icons/settings-circle.svg";
+import { ReactComponent as ConnectIcon } from "../../assets/icons/connect-circle .svg";
```

Comment:
Nit: I think there is a space before `.svg` in this file name. Maybe let's remove that to avoid some possible confusion when importing files in the future.